podman_compose.py

```diff
@@ -846,9 +850,9 @@ def _parse_compose_file(self):
                 "container-compose.override.yml",
                 "container-compose.override.yaml"
             ]))
-        files = args.file
+        files = args.file[0]
```

Comment:
I guess you would rather have me remove this; however I strongly believe this is the right thing to do as it could lead to unexpected results. That said, this discussion would be best for another PR, then :)

Reply:
yes, please remove this.